src/selectors/getRelativeSources.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import { getProjectDirectoryRoot, getSources } from "../selectors";
+import type { OuterState } from "../reducers/types";
+import type { Source } from "../types";
+import { dropScheme } from "../utils/source";
+
+export type RelativeSource = Source & {
+  +relativeUrl: string
+};
+
+function getRelativeUrl(url, root) {
+  if (!root) {
+    return dropScheme(url)
+      .split("/")
+      .slice(2)
+      .join("/");
```

Comment:
hmm, I was thinking it would be nice to include the domain/host... If we want to drop that too, it would be easier use `getSourcePath` or another helper...

Reply:
So if no root was set, you'd want the subtitle to be: "/davidwalsh.name/some/other/thing"?

Reply:
that's what I was thinking, because there could be several domains... does it matter, probably not.

I think I'm persuaded that the path is sufficient :)